sphinx/developers/java-library.rst

```diff
@@ -28,8 +31,8 @@ Alternatively Bio-Formats can be used by including its component jar files.
 You can :downloads:`download formats-gpl.jar <artifacts/formats-gpl.jar>` to
 use it as a library. Just add :file:`formats-gpl.jar` to your :envvar:`CLASSPATH` or
 build path. You will also need :file:`ome-common.jar` for common I/O functions,
-:file:`ome-xml.jar` for metadata standardization, and
-`SLF4J <http://www.slf4j.org/>`_ for :doc:`logging`.
+:file:`ome-xml.jar` for metadata standardization, and a
+`SLF4J <http://www.slf4j.org/>`_ binding for :doc:`logging`.
 
 Dependencies
```

Comment:
Should the table below indicate that logback is an optional dependency ?

Reply:
Yeah, and a lot of the versions are out of date, I will update the table